### PR TITLE
Feat/support area

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/components/Button/styles.module.css [hunk 1]
```diff
@@ -493,6 +493,13 @@ span.container {
   }
 }
 
+.borderMdS {
+  & .hotarea {
+    border-style: solid;
+    border-width: 1.25px;
```
Comment (robertu7) [inline src/components/Button/styles.module.css:499]: border with decimals width will be rounded to integer i think, please confirm.

File: src/components/Dialogs/SupportersDialog/Content.tsx [hunk 2]
```diff
@@ -87,21 +102,29 @@ const SupportersDialogContent = ({
...
-          ))}
+          <List
+            hasBorder={false}
+            spacing={isSmUp ? ['base', 'baseLoose'] : ['base', 'base']}
+          >
+            {edges.map(({ node, cursor }, i) => (
+              <List.Item key={cursor}>
+                <UserDigest.Rich
+                  user={node}
+                  key={cursor}
```
Comment (robertu7) [inline src/components/Dialogs/SupportersDialog/Content.tsx:113]: `key` is unnecessary here

File: src/components/Dialogs/SupportersDialog/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.count {
+  padding-left: var(--spacing-xx-tight);
+  font-weight: 590;
```
Comment (robertu7) [inline src/components/Dialogs/SupportersDialog/styles.module.css:3]: 500?

File: src/views/ArticleDetail/SupportWidget/Donators/index.tsx [hunk 4]
```diff
@@ -23,123 +16,91 @@ type AvatarItemPros = Pick<AvatarProps, 'user'> & {
...
 const AvatarItem = ({ user }: AvatarItemPros) => {
   return (
-    <Avatar
-      user={user || undefined}
-      src={user ? undefined : IMAGE_PIXEL}
-      size="lg"
-      title={user?.displayName || ''}
-    />
+    <Tooltip content={user?.displayName || ''} placement="top">
+      <div className={styles.avatarItem}>
```
Comment (robertu7) [inline src/views/ArticleDetail/SupportWidget/Donators/index.tsx:20]: should be clickable `<Link>` to user page?

**Review by robertu7 — APPROVED** [review]

**Comment by robertu7** [comment]
please link the issue to this PR

File: src/views/ArticleDetail/SupportWidget/Donators/index.tsx (hunk 4 above)
Comment (pitb2022) [reply src/views/ArticleDetail/SupportWidget/Donators/index.tsx:20]: https://www.figma.com/file/gs3w7fKdHqk9u1nnDxPYlZ?node-id=101:282&mode=dev#625981981
https://www.figma.com/file/gs3w7fKdHqk9u1nnDxPYlZ/%E4%BD%9C%E5%93%81%E9%A0%81?type=design&node-id=959-21502&mode=dev

**Comment by pitb2022** [comment]
https://github.com/thematters/developer-resource/issues/333

File: src/components/Button/styles.module.css (hunk 1 above)
Comment (pitb2022) [reply src/components/Button/styles.module.css:499]: Yes, just use 1px.

**Pull request merged**